Formal2024/Workshops/workshop2.lean

```diff
@@ -0,0 +1,239 @@
+import Mathlib.Data.Real.Basic
+import Mathlib.Tactic
+
+namespace Week2
+
+/-
+  # Logic: Implication, universal quantifier, existential quantifier, negation
+  References: [MIL] Mathematics in Lean, [Tut] Tutorials project.
+  Most of the demonstration section comes [MIL].
+-/
+
+/-!
+  # Implication and universal quantifier
+-/
+
+-- To write `∀` type
+
+-- Lean treats the universal quantifier as an implication.
+-- Implications are grouped on the RIGHT (N.B. this is different from the usual convention)
```

Comment:
> (N.B. this is different from the usual convention)

Isn't it the same though? If `P -> Q -> R`, then is the usual convention `(P -> Q) -> R`??

Reply:
The usual binary operations bind to the left: `1 + 2 + 3 = (1 + 2) + 3`. I suggest changing 'the usual convention' to 'operations like `+` and `*`.